bumble/transport/tcp_server.py

```diff
@@ -15,19 +15,25 @@
 # -----------------------------------------------------------------------------
 # Imports
 # -----------------------------------------------------------------------------
+from __future__ import annotations
 import asyncio
 import logging
 
-from .common import Transport, StreamPacketSource
+from .common import Transport, StreamPacketSource, BaseSink
 
 # -----------------------------------------------------------------------------
 # Logging
 # -----------------------------------------------------------------------------
 logger = logging.getLogger(__name__)
 
 
+class TcpServerTransport(Transport):
```

Comment:
Same remark as above: this should remain a function-local helper class, we don't want to leak the internal details of the implementation outside of the constructor.